API.md

```diff
@@ -60,6 +107,29 @@ fs.writeFileSync('./dest.min.json', format(style, 0));
 **Returns** `string`, stringified formatted JSON
 
 
+### `exports(layers)`
```

Comment:
Can you fix the naming like you did with `derefLayers`? Should be `groupByLayout`.